include/Oscillator.h

```diff
@@ -165,6 +233,25 @@ class LMMS_EXPORT Oscillator
 	float m_phaseOffset;
 	float m_phase;
 	const SampleBuffer * m_userWave;
+	bool m_useWaveTable;
+
+	/* Multiband WaveTable */
+	static sample_t s_waveTables[WaveShapes::NumWaveShapes-2][OscillatorConstants::WAVE_TABLES_PER_WAVEFORM_COUNT][OscillatorConstants::WAVETABLE_LENGTH];
```

Comment:
Instead I will add a new line `NumBasicWaveShapes = WhiteNoise` and use that for the wavetable stuff, since `NumWaveShapes` is also used by some instruments and they expect `NumWaveShapes - 1` to include the user waveform.